IntroComponentWithSign-upForm/script.js

```diff
@@ -0,0 +1,67 @@
+const form  = document.getElementById("form")
```

Comment:
Consistent spacing/formatting - Prettier again to the rescue!

```suggestion
const form = document.getElementById("form")
```